metals/src/main/scala/scala/meta/internal/metals/WorkspaceLspService.scala

```diff
@@ -157,20 +161,30 @@ class WorkspaceLspService(
           initTreeView,
           uri,
           name,
+          () => doctor.executeRefreshDoctor(),
+          executeRunDoctor,
```

Comment:
```suggestion
         doctor,
```
could we instead point to the main doctor? I would avoid passing functions because there are much harder to navigate.